app/models/internal/impl/PdfReportFormat.java

```diff
@@ -86,7 +87,9 @@ public int hashCode() {
         return Objects.hash(_widthInches, _heightInches);
     }
 
+    @JsonProperty("widthInches")
```

Comment:
Why is the internal model annotated for JSON serialization?

Reply:
Unintentional, reverted.